sarray_mode.py

```diff
@@ -0,0 +1,27 @@
+import graphlab as gl
+
+def mode_sa(sa, single_mode=True):
+    """Return a mode of sa, or all modes if there are several.
+
+    single_mode: whether to return a single mode or an SArray of all modes (default: True)."""
+
+    sf = gl.SFrame({"value": sa})
+    sf2 = sf.groupby("value", {"count": gl.aggregate.COUNT()})
+    max_count_index = sf2["count"].argmax()
+
+    if single_mode:
+        return sf2[max_count_index]["value"]
+
+    else:
+        max_count = sf2[max_count_index]["count"]
+        return sf2[sf2["count"] == max_count]["value"]
+
+# Create an SArray with two modes (most-common elements: 2 and 3)
+# sa = gl.SArray([1, 2, 2, 3, 3])
```

Comment:
Why is this commented-out? Should we uncomment it to make it clear that it's code you can run?